fdbserver/BlobManager.actor.cpp

```diff
@@ -5339,11 +5339,36 @@ ACTOR Future<Void> checkBlobManagerEpoch(Reference<AsyncVar<ServerDBInfo> const>
 	}
 }
 
+ACTOR Future<UID> fetchClusterId(Database db) {
+	loop {
+		state Reference<ReadYourWritesTransaction> tr = makeReference<ReadYourWritesTransaction>(db);
```

Comment:
the transaction should be created outside of the loop so that onError has proper backoff state

Reply:
fixed. thank you!